v2/cmd/integration-test/integration-test.go

```diff
@@ -52,6 +67,18 @@ func main() {
 	}
 }
 
+func debugTests() {
+	for tpath, testcase := range protocolTests[runProtocol] {
+		if runTemplate != "" && !strings.Contains(tpath, runTemplate) {
+			continue
+		} else {
```

Comment:
I think the else here is not necessary since the previous branch has a `continue`